mwclient/client.py

```diff
@@ -1349,7 +1623,9 @@ def recentchanges(self, start=None, end=None, dir='older', namespace=None,
         return listing.List(self, 'recentchanges', 'rc', max_items=max_items,
                             api_chunk_size=api_chunk_size, **kwargs)
 
-    def revisions(self, revids, prop='ids|timestamp|flags|comment|user'):
+    def revisions(
+        self, revids: List[int], prop: str = 'ids|timestamp|flags|comment|user'
```

Comment:
I think `revids` could be ints or strs, since we actually force them to str in the function body anyway.

Reply:
Changed.